README.md

```diff
@@ -334,6 +334,62 @@ following keys:
 
 In both the input and output all keys should be symbolized.
 
+Connecting Webhooks
+===================
+
+Many content providers such as Contentful or Prismic can send payloads via webhooks when content changes, those can be very useful in a number of ways.
+
+We suggest using the [`solidus_webhooks`](http://github.com/solidusio-contrib/solidus_webhooks#readme) extension to take the most out of `solidus_content`, let's see some examples.
+
+Add this to your Gemfile:
+
+```rb
+gem "solidus_webhooks"
+```
+
+
+### Using Webhooks to Auto-Create entries
```

Comment:
I don't know if this is a useful example. This is not how I'd recommend for people to interact with Contentful: you should be calling the Contentful API directly when you need information, not retain a list of all the entries in Contentful and try to keep it in sync.

Reply:
I see your point about the validity of the on-demand approach, but I fail to get the downsides of actively keeping content in in-sync. Can you please elaborate a bit on that?

Reply:
@elia yep, sorry! I guess the main downside is you're introducing complexity by trying to keep your data in sync with Contentful, and things may easily go wrong (e.g. the webhook doesn't get called when you expect it to be). The same can be said about simply using the webhook to invalidate the cache of course, but by using it to keep track of new entries and their data, you're increasing the error surface.

In any case, if you feel strongly about this I'm happy to keep it in as it's not technically an error, but I'm curious to hear your thoughts!

Reply:
Yeah, thanks for explaining, those are clearly contrived examples and until someone tries them IRL discussing is the best we can do. 

About this specifically I think, all things considered, I'd like to keep it in the readme. On one side I think it's really convenient if someone manages to skip the repetitiveness of manually creating the entries, on the other the kind of risk assessment you highlighted is the reason for which is better to keep them as examples rather than providing "officially supported" code.